themes/blist/assets/css/styles.css

```diff
@@ -1,10 +1,54 @@
 /* Tailwind base - put variables under: tailwind.config.js */
-@import "node_modules/tailwindcss/base";
+@import 'node_modules/tailwindcss/base';
 /* Tailwind component classes registered by plugins*/
-@import "node_modules/tailwindcss/components";
+@import 'node_modules/tailwindcss/components';
 /* Site Specific */
-@import "assets/css/site";
+@import 'assets/css/site';
 /* Paginator */
-@import "assets/css/paginator";
+@import 'assets/css/paginator';
 /* Tailwind's utility classes - generated based on config file */
-@import "node_modules/tailwindcss/utilities";
\ No newline at end of file
+@import 'node_modules/tailwindcss/utilities';
+
+input,
```

Comment:
This theme is using Tailwindcss lib. It's better to use classes from this library.
Here is an example.
https://tailwindui.com/components/application-ui/forms/input-groups